config/webpack.dev.config.js

```diff
@@ -110,6 +110,7 @@ module.exports = Merge.smart(commonConfig, {
       REFRESH_ACCESS_TOKEN_ENDPOINT: 'http://localhost:18000/login',
       DATA_API_BASE_URL: 'http://localhost:8000',
       ECOMMERCE_BASE_URL: 'http://localhost:18130',
+      LICENSE_MANAGER_BASE_URL: 'http://localhost:18170',
```

Comment:
We'll also need to add `LICENSE_MANAGER_BASE_URL` to edx-internal so that stage/prod are aware of it.

Reply:
Sure, I can take care of that. What's the prod/stage URL for license manager?

Reply:
Here's the PR: https://github.com/edx/edx-internal/pull/2390